### PR TITLE
GET dispute scheme files endpoint

### PR DESCRIPTION
**Pull request by konstantinos-kafkas-cko**
# Description of changes in PR

https://checkout.atlassian.net/browse/DI-709

Introduces the specs for the newly introduced dispute scheme file endpoint to both MBC and NAS websites. This PR won't be merged until the BETA testing is passing  which means the selected merchants tested it and approved it

GET dispute scheme files endpoint: `https://api.checkout.com/disputes/dsp_rbhwd2qrg13uhrp2newf/schemefiles`

Preview ABC: https://checkout-api-reference-git-scheme-files-cko-frontend.vercel.app/
Preview NAS: https://checkout-api-reference-git-scheme-files-cko-frontend.vercel.app/preview/crusoe

## Checklist

- [x] Added a changelog entry on the `swagger.yaml` file
- [x] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Review by konstantinos-mastropavlos-cko — APPROVED** [review]
This is now RELEASED in PRODUCTION - Let's update the docs! ⭐

File: abc_spec/paths/disputes@{dispute_id}@schemefiles.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+get:
+  security:
+    - ApiSecretKey: [ ]
+  tags:
+    - Disputes
+  summary: Get dispute scheme files (BETA)
```
Comment (sebastian-garcia-cardona-cko) [inline abc_spec/paths/disputes@{dispute_id}@schemefiles.yaml:6]:
```suggestion
  summary: Get dispute scheme files
```

File: abc_spec/paths/disputes@{dispute_id}@schemefiles.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  operationId: getDisputeSchemeFiles
+  description: Returns all the scheme files of a dispute using the dispute identifier.
```
Comment (sebastian-garcia-cardona-cko) [inline abc_spec/paths/disputes@{dispute_id}@schemefiles.yaml:8]:
```suggestion
  description: |
    <span style="font-size: 14px; font-weight: 500; color: rgb(60, 4, 133); background-color: rgb(237, 222, 255); border-radius: 4px; padding: 2px 6px" >Beta</span><p></p>
    Returns all of the scheme files of a dispute using the dispute identifier.
```

File: nas_spec/paths/disputes@{dispute_id}@schemefiles.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+get:
+  security:
+    - OAuth:
+        - disputes
+        - disputes:scheme-files
+    - ApiSecretKey: [ ]
+  tags:
+    - Disputes
+  summary: Get dispute scheme files (BETA)
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/paths/disputes@{dispute_id}@schemefiles.yaml:9]:
```suggestion
  summary: Get dispute scheme files
```

File: nas_spec/paths/disputes@{dispute_id}@schemefiles.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  operationId: getDisputeSchemeFiles
+  description: Returns all the scheme files of a dispute using the dispute identifier.
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/paths/disputes@{dispute_id}@schemefiles.yaml:11]:
```suggestion
  description: |
    <span style="font-size: 14px; font-weight: 500; color: rgb(60, 4, 133); background-color: rgb(237, 222, 255); border-radius: 4px; padding: 2px 6px" >Beta</span><p></p>
    Returns all of the scheme files of a dispute using the dispute identifier.
```

Comment (sebastian-garcia-cardona-cko) [inline nas_spec/paths/disputes@{dispute_id}@schemefiles.yaml:11]:
```suggestion
  description: |
    <span style="font-size: 14px; font-weight: 500; color: rgb(60, 4, 133); background-color: rgb(237, 222, 255); border-radius: 4px; padding: 2px 6px" >Beta</span><p></p>
    Returns all of the scheme files of a dispute using the dispute identifier.
```

**Review by sebastian-garcia-cardona-cko — CHANGES_REQUESTED** [review]

**Review by sebastian-garcia-cardona-cko — DISMISSED** [review]

**Comment by konstantinos-mastropavlos-cko** [comment]
@sebastian-garcia-cardona-cko reverted previously committed BETA, as the feature is perceived as final - let me know if any thoughts 🙏🏼

**Review by konstantinos-mastropavlos-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Pull request merged**